iroh-util/src/lib.rs

```diff
@@ -18,6 +18,7 @@ use dirs::home_dir;
 use tracing::debug;
 
 const IROH_DIR: &str = ".iroh";
+const DEFAULT_NOFILE_LIMIT: u64 = 1 << 16;
```

Comment:
an we please use regular numbers not shifts, they are just harder to read, eg 1024 * 1024 is much easier to read